src/components/strategies/create/useCreateStrategyTypeMenu.tsx

```diff
@@ -51,99 +55,117 @@ export const useCreateStrategyTypeMenu = (
-    },
-  ];
+    ],
+    [base, quote]
+  );
 
-  const typeDisposable: StrategyTypeItemSvg[] = [
-    {
-      label: 'Buy Limit',
-      svg: <IconBuyLimit className={'w-60 text-green'} />,
-      to: PathNames.createStrategy,
-      search: {
-        base,
-        quote,
-        strategyType: 'disposable',
-        strategyDirection: 'buy',
-        strategySettings: 'limit',
+  const typeDisposable: StrategyTypeItemSvg[] = useMemo(
+    () => [
+      {
+        label: 'Buy Limit',
+        svg: <IconBuyLimit className={'w-60 text-green'} />,
+        to: PathNames.createStrategy,
+        search: {
+          base,
+          quote,
+          strategyType: 'disposable',
+          strategyDirection: 'buy',
+          strategySettings: 'limit',
+        },
+        isRecommended: true,
       },
-    },
-    {
-      label: 'Buy Range',
-      svg: <IconBuyRange className={'w-60 text-green'} />,
-      to: PathNames.createStrategy,
-      search: {
-        base,
-        quote,
-        strategyType: 'disposable',
-        strategyDirection: 'buy',
-        strategySettings: 'range',
+      {
+        label: 'Sell Limit',
+        svg: <IconSellLimit className={'w-60 text-red'} />,
+        to: PathNames.createStrategy,
+        search: {
+          base,
+          quote,
+          strategyType: 'disposable',
+          strategyDirection: 'sell',
+          strategySettings: 'limit',
+        },
+        isRecommended: true,
       },
-    },
-    {
-      label: 'Sell Limit',
-      svg: <IconSellLimit className={'w-60 text-red'} />,
-      to: PathNames.createStrategy,
-      search: {
-        base,
-        quote,
-        strategyType: 'disposable',
-        strategyDirection: 'sell',
-        strategySettings: 'limit',
+      {
+        label: 'Buy Range',
+        svg: <IconBuyRange className={'w-60 text-green'} />,
+        to: PathNames.createStrategy,
+        search: {
+          base,
+          quote,
+          strategyType: 'disposable',
+          strategyDirection: 'buy',
+          strategySettings: 'range',
+        },
       },
-    },
-    {
-      label: 'Sell Range',
-      svg: <IconSellRange className={'w-60 text-red'} />,
-      to: PathNames.createStrategy,
-      search: {
-        base,
-        quote,
-        strategyType: 'disposable',
-        strategyDirection: 'sell',
-        strategySettings: 'range',
+      {
+        label: 'Sell Range',
+        svg: <IconSellRange className={'w-60 text-red'} />,
+        to: PathNames.createStrategy,
+        search: {
+          base,
+          quote,
+          strategyType: 'disposable',
+          strategyDirection: 'sell',
+          strategySettings: 'range',
+        },
       },
-    },
-  ];
+    ],
+    [base, quote]
+  );
 
   const handleClick = (
     to: string,
     search: StrategyCreateLocationGenerics['Search'],
     replace?: boolean
   ) => {
-    navigate({ to, search, replace });
+    if (
+      search.strategySettings !== 'limit' &&
```

Comment:
I noticed that in CreateStrategyOrders.tsx you have the opposite condition: 
strategySettings === 'range' || strategySettings === 'custom'
I think it would be better to pick one style and use it in both places . same result but a little bit easier to read.